Doc/reference/executionmodel.rst

```diff
@@ -139,8 +139,9 @@ namespace.  Names are resolved in the top-level namespace by searching the
 global namespace, i.e. the namespace of the module containing the code block,
 and the builtins namespace, the namespace of the module :mod:`builtins`.  The
 global namespace is searched first.  If the names are not found there, the
-builtins namespace is searched.  The :keyword:`!global` statement must precede
-all uses of the listed names.
+builtins namespace is searched next.  If the names are also not found in the
+builtins namespace, new variables are created in the global namespace.
+The global statement must precede all uses of the listed names.
```

Comment:
```suggestion
builtins namespace is searched next. If the names are also not found in the
builtins namespace, new variables are created in the global namespace.
The :keyword:`global` statement must precede all uses of the listed names.
```

Reply:
There is a link to global keyword in the next sentence(start of a new paragraph). There is also a link to global keyword in the first sentence on the same paragraph.

All in all, there are three links to global keywords within a very short span - hence I proposed removing this one.